handouts/2020-02-04-diversity-blurb.txt

```diff
@@ -0,0 +1,22 @@
+
+There is no such thing as a meritocracy. An unchecked imbalance of power will crystallize to become a structure. Power is weaponized in the duality of privilege and discrimination but all three are invisible. If deep, structural power imbalances exist we have gone off course. We have gone off course.
```

Comment:
This is a stylistic critique, so it's a bit hard to deliver this. I'm picking off sentences, since I found that easiest.

Out of the 5 sentences in the paragraph, the second and third sentence is a tad bit difficult to sift through. I like how it opens and ends, but I had to re-read the two in-between a couple of times to get it.

I know [Ankur](https://twitter.com/ankurs3thi) has some contempt for content that is even borderline poetic – and this is not quite that – but if you could see if you make it easier to read and parse in the first attempt, that would probably be easier for people to digest in a conference-type setting.

Reply:
Yeah, this was mostly an effort to maintain density. I can break this apart a bit but the word count will go up.